package.json

```
@@ -61,13 +61,13 @@
 		"shorten-repo-url": "^2.0.0",
 		"strip-indent": "^3.0.0",
 		"text-field-edit": "^3.1.0",
-		"tiny-version-compare": "2.0.0",
+		"tiny-version-compare": "3.0.0",
```

v2 is ESM, it's still preferable. For v3 I had to revert to CJS

```suggestion
		"tiny-version-compare": "2.0.0",
```

Notice the missing `^`, it means this version should not be changed

---

Your right I am not sure why I updated it...